config/packages/security.yaml

```diff
@@ -1,13 +1,30 @@
 security:
+    encoders:
+        bcrypt:
+            algorithm: bcrypt
+            cost: 12
     providers:
         users:
             id: 'App\Infrastructure\User\Auth\AuthProvider'
-                
+
     firewalls:
         dev:
             pattern: ^/(_(profiler|wdt)|css|images|js)/
             security: false
 
+        api_auth:
```

Comment:
`api_auth` -> `api_public` maybe?

Reply:
`api_auth` if you would ask me (more expressive)